src/main/java/org/spongepowered/common/mixin/core/tileentity/MixinTileEntityMobSpawner.java

```diff
@@ -58,6 +59,6 @@ public void spawnEntityBatchImmediately(boolean force) {
     @Override
     public void supplyVanillaManipulators(List<DataManipulator<?, ?>> manipulators) {
         super.supplyVanillaManipulators(manipulators);
-        // TODO manipulators.add(getMobSpawnerData());
+        manipulators.add(getMobSpawnerData());
```

Comment:
How do you suggest I do that? Directly create the SpongeMobSpawnerData class? I looked at a couple other entities and couldn't find an example implementation.